Mathlib/RingTheory/WittVector/FrobeniusFractionField.lean

```diff
@@ -242,8 +242,6 @@ theorem exists_frobenius_solution_fractionRing_aux (m n : ℕ) (r' q' : 𝕎 k)
       (IsFractionRing.injective (𝕎 k) (FractionRing (𝕎 k))).ne hq'''
   rw [zpow_sub₀ (FractionRing.p_nonzero p k)]
   field_simp [FractionRing.p_nonzero p k]
-  simp only [IsFractionRing.fieldEquivOfRingEquiv, IsLocalization.ringEquivOfRingEquiv_eq,
-    RingEquiv.coe_ofBijective]
```

Comment:
What happened here?

Reply:
In the absence of API, the alternative was to simply unfold the definitions until simp could take care of it. With the API, this is no longer necessary.